### PR TITLE
Disable OCSP direct pushing during upgrade

### PR DESCRIPTION
**Pull request by fmarco76**
The direct publishing to OCSP is not working properly and a previous commit has change the default value for the `ca.publish.rule.instance.ocsprule-<instance-<port>.enable` attribute to false. This commit add the upgrade script to set false for the existing instances during the upgrade.

There are no problems with existing instances because the communication with OCSP was not properly working and other mechanism were in place.

Close the issue: RHCS-4085

### CODE REVIEW

**Review by ladycfu — APPROVED** [review]
LGTM if tested to work.

**Comment by edewata** [comment]
I think the code looks good, but PKI 10.13.5 has been tagged and built, so the `v10.13` branch actually contains the code meant for 10.13.6. I think we should bump the version number in the spec file and move this script into 10.13.6 folder. @ckelleyRH What do you think?

BTW, here's some doc on writing an upgrade script: https://github.com/dogtagpki/pki/blob/master/docs/development/Creating-Upgrade-Script.adoc

**Comment by fmarco76** [comment]
> I think the code looks good, but PKI 10.13.5 has been tagged and built, so the `v10.13` branch actually contains the code meant for 10.13.6. I think we should bump the version number in the spec file and move this script into 10.13.6 folder. @ckelleyRH What do you think?
> 

Updated the folder to the next version.

> BTW, here's some doc on writing an upgrade script: https://github.com/dogtagpki/pki/blob/master/docs/development/Creating-Upgrade-Script.adoc

That very useful, it make the manual  test much easier. Thanks!

**Review by ckelleyRH — APPROVED** [review]
Apparently line 32 contains whitespace that is causing the `pylint` job to fail (I can't see it in the diff in the GUI though, but maybe it is auotmagically stripped?). Once that is resolved it is good to merge for me.

**Comment by fmarco76** [comment]
@ladycfu @edewata @ckelleyRH Thanks!

**Pull request merged**
